cli/internal/plugins/bzr/versioner.go

```diff
@@ -0,0 +1,79 @@
+package bzr
+
+import (
+	"bytes"
+	"context"
+	"os/exec"
+	"strings"
+
+	"github.com/gobuffalo/buffalo-cli/plugins"
+)
+
+// Versioner ...
+type Versioner struct {
+	pluginsFn plugins.PluginFeeder
+}
+
+// WithPlugins ...
+func (b *Versioner) WithPlugins(f plugins.PluginFeeder) {
+	b.pluginsFn = f
+}
+
+// ScopedPlugins ...
+func (b *Versioner) ScopedPlugins() []plugins.Plugin {
+	if b.pluginsFn == nil {
+		return []plugins.Plugin{}
+	}
+
+	var scoped []plugins.Plugin
+	for _, p := range b.pluginsFn() {
+		switch p.(type) {
+		case CommandRunner:
+			scoped = append(scoped, p)
+		}
+	}
+
+	return scoped
+}
+
+// BuildVersion is used by other commands to get the build
+// version of the current source and use it for the build.
+func (b *Versioner) BuildVersion(ctx context.Context, root string) (string, error) {
+
+	cmd := exec.CommandContext(ctx, "bzr", "revno")
+	bb := &bytes.Buffer{}
+	cmd.Stdout = bb
+
+	var fn cmdRunnerFn = func(ctx context.Context, cmd *exec.Cmd) error {
+		if _, err := exec.LookPath("bzr"); err != nil {
```

Comment:
Remove the check, as its unneeded. when you try to run the command it'll return an error if it can't find it.